ui/raidboss/popup-text.ts

```diff
@@ -762,7 +762,7 @@ export class PopupText {
               trigger.localNetRegex = Regexes.parse(localeNetRegex);
               orderedTriggers.push(trigger);
               found = true;
-            } else if (defaultNetRegex) {
+            } else if (defaultNetRegex !== undefined) {
```

Comment:
this trigger eslint error about "always true"

Reply:
we may need to refactor the code for processing triggers, it would be much easy to maintain if it's pure function.

Reply:
And I find the reason why it's not working:

```ts
export type LooseTriggerSet =
  & Exclude<Partial<TriggerSet<RaidbossData>>, 'triggers' | 'timelineTriggers'>
  & {
    /** @deprecated Use zoneId instead */
    zoneRegex?:
      | RegExp
      | { [lang in Lang]?: RegExp };
    triggers?: LooseTrigger[];
    timelineTriggers?: LooseTimelineTrigger[];
  };
```

here it's not `Exclude`, it should be `Omit`